paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/SavedPaymentMethodRowButtonScreenshotTest.kt

```diff
@@ -0,0 +1,279 @@
+            code = PaymentMethod.Type.SepaDebit.code,
+            type = PaymentMethod.Type.SepaDebit,
+        )
+    )
+
+    private val savedUsBankAccount =
+        DisplayableSavedPaymentMethod(
+            displayName = "4242",
+            paymentMethod = PaymentMethod(
+                id = "002",
+                created = null,
+                liveMode = false,
+                code = PaymentMethod.Type.USBankAccount.code,
+                type = PaymentMethod.Type.USBankAccount,
+                usBankAccount = PaymentMethodFixtures.US_BANK_ACCOUNT.usBankAccount,
+            )
+        )
+
+    private val savedOtherPaymentMethod = DisplayableSavedPaymentMethod(
+        displayName = "Klarna",
+        paymentMethod = PaymentMethod(
+            id = "002",
+            created = null,
+            liveMode = false,
+            code = PaymentMethod.Type.Klarna.code,
+            type = PaymentMethod.Type.Klarna,
+        )
+    )
+
+    @Test
+    fun testSavedVisa() {
+        paparazziRule.snapshot {
+            SavedPaymentMethodRowButton(
+                displayableSavedPaymentMethod = savedVisa,
+                resources = null,
+                isEnabled = true,
+                isSelected = false,
+            )
+        }
+    }
+
+    @Test
+    fun testSavedVisa_selected() {
+        paparazziRule.snapshot {
+            SavedPaymentMethodRowButton(
+                displayableSavedPaymentMethod = savedVisa,
+                resources = null,
+                isEnabled = true,
+                isSelected = true,
+            )
+        }
+    }
+
+    @Test
+    fun testSavedVisa_disabled() {
+        paparazziRule.snapshot {
+            SavedPaymentMethodRowButton(
+                displayableSavedPaymentMethod = savedVisa,
+                resources = null,
+                isEnabled = false,
+                isSelected = false,
+            )
+        }
+    }
+
+    @Test
+    fun testSavedUsBankAccount() {
+        paparazziRule.snapshot {
+            SavedPaymentMethodRowButton(
+                displayableSavedPaymentMethod = savedUsBankAccount,
+                resources = null,
+                isEnabled = true,
+                isSelected = false,
+            )
+        }
+    }
+
+    @Test
+    fun testSavedUsBankAccount_selected() {
+        paparazziRule.snapshot {
+            SavedPaymentMethodRowButton(
+                displayableSavedPaymentMethod = savedUsBankAccount,
+                resources = null,
+                isEnabled = true,
+                isSelected = true,
+            )
+        }
+    }
+
+    @Test
+    fun testSavedUsBankAccount_disabled() {
+        paparazziRule.snapshot {
+            SavedPaymentMethodRowButton(
+                displayableSavedPaymentMethod = savedUsBankAccount,
+                resources = null,
+                isEnabled = false,
+                isSelected = false,
+            )
+        }
+    }
+
+    @Test
+    fun testSavedAmex() {
```

Comment:
I don't think we should screenshot each card type (especially in every variant).

Reply:
updated to not include the variants